docs/Versioning.md

```diff
@@ -71,7 +71,9 @@ A given operator is identified by a three-tuple: `(domain, op_type, and op_versi
 
 * Adding/removing/reordering inputs or outputs.
 
-* Adding/removing types supported by inputs and outputs, and changing types used by attributes.
+* Removing types supported by inputs or outputs.
```

Comment:
why bump up version for removing types? do we need to treat add and remove different?

Reply:
Because removing type can break existing implementation.